examples/src/demo.cr

```diff
@@ -35,7 +35,7 @@ MY_APP_SERVER.config do |app|
 
   # All static content will run these transformations
   pipeline :static do
-    plug HTTP::StaticFileHandler.new "../examples/public", true
+    plug HTTP::StaticFileHandler.new "../examples/public", false
```

Comment:
`false` stops from processing the next handler, `true` allows for the processing of the handler.

Doesn't really matter this is is just for demonstration.